Tests/js/views/fields/assets/ez-selection-editview-tests.js

```diff
@@ -597,11 +597,11 @@ YUI.add('ez-selection-editview-tests', function (Y) {
             },
 
             _assertCorrectFieldValue: function (fieldValue, msg) {
-                Y.Assert.areEqual(options.indexOf(this.newValue[0]), fieldValue[0], msg);
-                Y.Assert.areEqual(options.indexOf(this.newValue[1]), fieldValue[1], msg);
+                Y.Assert.areEqual(options.indexOf(this.newValue[0].text), fieldValue[0], msg);
+                Y.Assert.areEqual(options.indexOf(this.newValue[1].text), fieldValue[1], msg);
             },
 
-            newValue: ['Vergongeat', 'Croix Rousse']
+            newValue: [{text: 'Vergongeat'}, {text:'Croix Rousse'}]
```

Comment:
weird to have to make this change, normally from an outside point of view, SelectionEditView should not have been changed